### PR TITLE
Added the real elements

### PR DESCRIPTION
**Pull request by eanker**
Added the real elements to the repo.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
   ],
   "repository": {
     "type": "git",
-    "url": "git://github.com/preview-code/comment-container.git"
+    "url": "git://github.com/preview-code/comments.git"
   },
-  "homepage": "https://github.com/preview-code/comment-container",
-  "main": "comment-container.html",
+  "homepage": "https://github.com/preview-code/comments",
+  "main": "comment-list.html",
```
Comment (txsmith) [inline bower.json]: Since we have multiple `html` files, the main can be removed

File: bower.json [hunk 2]
```diff
@@ -21,20 +21,31 @@
...
     "comment",
     "review",
     "code"
-  ]
-}
\ No newline at end of file
+  ],
+  "resolutions": {
+    "iron-test-helpers": "^1.0.0"
+  }
```
Comment (txsmith) [inline bower.json]: Resolutions is not needed

File: demo/comment/comment.html [hunk 3]
```diff
@@ -16,7 +16,7 @@
     </style>
 
  <comment-list list="[[comments]]"></comment-list>
- <add-comment hidden="{{hideAdd}}" comments="{{comments}}", user="[[user]]", line-comment="[[lineComment]]"></add-comment>
+ <add-comment hidden="{{hideAdd}}" comments="{{comments}}" line-comment="[[lineComment]]" project="[[project]]"></add-comment>
```
Comment (txsmith) [inline demo/comment/comment.html]: `lineComment` seems to be undefined

**Comment by eanker** [comment]
Fixes #1 due to changes upstream.

**Pull request merged**